### ISSUE TITLE
Canary binary is missing

### ISSUE THREAD

**Issue by willemneal** [AssemblyScript/assemblyscript#989]
Currently get the following CI error:

```
curl: (22) The requested URL returned error: 404 
Binary download from https://nodejs.org/download/v8-canary//v14.0.0-v8-canary2019120435eb057712/node-v14.0.0-v8-canary2019120435eb057712-linux-x64.tar.xz failed, trying source.
grep: /home/runner/.nvm/.cache/bin/node-v14.0.0-v8-canary2019120435eb057712-linux-x64/node-v14.0.0-v8-canary2019120435eb057712-linux-x64.tar.xz: No such file or directory
Provided file to checksum does not exist.
Binary download failed, trying source.
/home/runner/work/_actions/dcodeIO/setup-node-nvm/v1.0.0/index.js:16
  if (code !== 0) throw Error("installation failed with " + code);
                  ^

Error: installation failed with 1
    at ChildProcess.<anonymous> (/home/runner/work/_actions/dcodeIO/setup-node-nvm/v1.0.0/index.js:16:25)
    at ChildProcess.emit (events.js:200:13)
    at maybeClose (internal/child_process.js:1021:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
```

Where do I update the url to the binary?

**Comment by dcodeIO** [comment]
There are [some](https://nodejs.org/download/v8-canary/v14.0.0-v8-canary2019120435eb057712/) binaries but not others it seems. Wondering if it is still building or failed. Suggesting to ignore the canary build for now.

**Comment by dcodeIO** [comment]
Looks like they have [issues](https://github.com/nodejs/node-v8/issues).

**Comment by willemneal** [comment]
Okay cool thanks!

**Comment by dcodeIO** [comment]
The issues appear like those might take a while to resolve, so perhaps pinning to `v14.0.0-v8-canary201911242015a12d82` is indeed an option? Hmm

**Comment by MaxGraey** [comment]
I guess pinning for now is better then completely disable

**Comment by MaxGraey** [comment]
Solved in #990
